deploy/029_deploy_linea_spokepool.ts

```diff
@@ -14,11 +14,16 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     1_000_000,
     L2_ADDRESS_MAP[chainId].lineaMessageService,
     L2_ADDRESS_MAP[chainId].lineaTokenBridge,
-    L2_ADDRESS_MAP[chainId].lineaUsdcBridge,
     hubPool.address,
     hubPool.address,
   ];
-  const constructorArgs = [WETH[chainId], QUOTE_TIME_BUFFER, FILL_DEADLINE_BUFFER];
+  const constructorArgs = [
+    WETH[chainId],
+    QUOTE_TIME_BUFFER,
+    FILL_DEADLINE_BUFFER,
+    USDC[chainId],
```

Comment:
Same as above - I think _for now_ we need `USDCe` instead.

Reply:
f908e3888dd80f1f7c8621966edcbd5de9aae7fe